scripts/CP20-JP/c100266010.lua

```diff
@@ -0,0 +1,57 @@
+--Fossil Dragon Skullgar
+--Scripted by: XGlitchy30
+local function getID()
```

Comment:
why use this...

Reply:
I must have forgotten to erase that function from the script. I'm used to write scripts with this getID() function, then, after testing, I replace all the id/cid occurrences. Sorry for the inconvenience.